integration_tests/tests/vault/update_token_metadata.rs

```diff
@@ -1,124 +1,191 @@
 #[cfg(test)]
```

Comment:
I think some of these tests are mixing up update_vault_balance with update_token_metadata?

Reply:
Oh boo, you're correct. Thanks for catching that.

Reply:
fixed